tests/test_preproc.py

```diff
@@ -994,7 +994,7 @@ def test_parse_bmi3d_v14(self):
         self.check_required_fields(data, metadata)
         files['ecube'] = '2023-12-29_BMI3D_te13154'
         data, metadata = parse_bmi3d(data_dir, files) # with ecube data
-        self.assertEqual(len(data['sync_events']), len(data['bmi3d_events']))
+        #self.assertEqual(len(data['sync_events']), len(data['bmi3d_events']))
```

Comment:
I got this error related to this line, but I didn't know how to remove error. I commented out this line for now. Do you think this is important?
![image](https://github.com/user-attachments/assets/1ccb2eae-8979-44f7-942b-5ea1d180e960)

Reply:
This was because of my local environment. Now I uncommented the line.

Reply:
I don't understand why testing in local gives different results from testing in github. I'll ask you about this when I come back.